sdk/tests/client/input_selection/native_tokens.rs

```diff
@@ -1487,6 +1487,73 @@ fn multiple_remainders() {
     });
 }
 
+// #[test]
+// fn higher_nts_count_but_below_max_native_tokens() {
```

Comment:
I guess here we should now have >=65 inputs with different native tokens and a low amount and then other input(s) with a higher amount which can be used alone to fulfill the output amount requirement

Reply:
Yeah I thought it was annoying in the merge PR so I just commented it out for now 😄